src/tmxrasterizer/tmxrasterizer.qbs

```diff
@@ -8,7 +8,7 @@ QtGuiApplication {
     Depends { name: "libtiled" }
 
     cpp.includePaths: ["."]
-    cpp.rpaths: ["$ORIGIN/../lib"]
+    cpp.rpaths: qbs.targetOS.contains("darwin") ? ["@loader_path/../Frameworks"] : ["$ORIGIN/../lib"]
```

Comment:
It's the responsibility of the caller to specify where it looks for its libraries, not the callee.

The fact that libtiled specifies where it's installed, rather than Tiled.app specifying libtiled as an bundle-embedded library dependency, is currently a limitation of Qbs.